pyproject.toml

```diff
@@ -111,6 +111,7 @@ awadb = {version = "^0.3.3", optional = true}
 azure-search-documents = {version = "11.4.0a20230509004", source = "azure-sdk-dev", optional = true}
 openllm = {version = ">=0.1.6", optional = true}
 streamlit = {version = "^1.18.0", optional = true, python = ">=3.8.1,<3.9.7 || >3.9.7,<4.0"}
+fastapi-poe = {version = "^0.0.14", optional = true}
```

Comment:
will need to add this to a group as well (like `all`) or poetry will make it required when building the package